src/shared/mkfs-util.c

```diff
@@ -140,6 +140,12 @@ static int do_mcopy(const char *node, const char *root) {
         if (dir_is_empty(root, /*ignore_hidden_or_backup=*/ false))
                 return 0;
 
+        r = find_executable("mcopy", NULL);
```

Comment:
if you use find_executable() anyway, you might as well use the return value as first argument to the strv_new() call below.